src/contraction/contractionHierarchies_driver.cpp

```
@@ -0,0 +1,135 @@
+
+#include "cpp_common/pgdata_getters.hpp"
+#include "cpp_common/to_postgres.hpp"
+#include "contraction/ch_graphs.hpp"
+#include "contraction/contractionHierarchies.hpp"
+
+#include "c_types/contractionHierarchies_rt.h"
+#include "cpp_common/identifiers.hpp"
+#include "cpp_common/alloc.hpp"
+
+void
+pgr_contractionHierarchies(
```

Cppcheck (2.10-2)

[style] The function 'pgr_contractionHierarchies' is never used.

(unusedFunction)